Curve-finance-suite/Create-lockevent/src/agent.spec.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+} from 'forta-agent';
+import {
+  provideCreateLockAgent, 
+} from './agent';
+import { 
+  encodeFunctionSignature, 
+  TestTransactionEvent, 
+  createAddress,
+  encodeParameter,
+  encodeParameters
+} from "forta-agent-tools";
+
+const CONTRACT_ADDRESS = createAddress('0x1');
+const ALERT_ID = 'Create lock Agent test';
+const SENDER_ADDRESS = createAddress('0x5');
+const EVENT_SIGNATURE = 'Deposit(address,uint256,uint256,int128,uint256)';
+
+const createFinding = (from: string, value:string, locktime: string ) => Finding.fromObject({
+  name: 'Lock creation on Voting Escrow contract Detected',
+  description: 'Deposit event with type 1 was emitted',
+  alertId: ALERT_ID,
+  severity: 2,
+  type: 2,
```

Comment:
Can we use the constants from `forta-sdk` for making this more readable?
`FindingSeverity.Low` and `FindingType.Suspicious`